### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by karniv00l**

### CODE REVIEW

**Comment by akondas** [comment]
Small fixes ... 42 changes :thinking: ...

File: templates/component/js/ga.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+(function() {
+    window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
+    ga('create', '{{ ga_tracking }}', 'auto');
+    ga('send', 'pageview');
+
+    if (isOAuthReferrer(document.referrer)) {
+        ga('set', 'referrer', '{{ url("index") }}');
```
Comment (karniv00l) [inline templates/component/js/ga.js:7]: this sets GA referrer manually to index route for oauth redirections

File: templates/organization/stats.html.twig [hunk 2]
```diff
@@ -1,23 +1,16 @@
...
 
-{% block header %}{{ organization.name }} package installs (last 30 days){% endblock %}
+{% set days = days ?? 30 %}
+
+{% block header %}{{ organization.name }} package installs (last {{ days }} days){% endblock %}
 
 {% block header_btn %}
-{% set days = days ?? 30 %}
-    <select class="form-control period-select">
+    <select class="form-control period-select selectpicker" data-style="btn-secondary">
```
Comment (karniv00l) [inline templates/organization/stats.html.twig:8]: Screenshot 2020-05-06 at 18 48 54

**Comment by karniv00l** [comment]
nothing major, mainly removed some of the duplications in templates

**Pull request merged**